core-services/prow/02_config/maistra/maistra-test-tool/_prowconfig.yaml

```diff
@@ -0,0 +1,12 @@
+tide:
+  queries:
+  - labels:
+    - auto-merge
```

Comment:
we don't need this label in this section

Reply:
I removed the auto-merge label in prowconfig yaml files